guides/navplace.md

```diff
@@ -0,0 +1,298 @@
+            {
+              attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+              maxZoom: 19
+            }
+            /**
+              Define the base map projection of the Earth that you want (satellite, elevation, infrared, etc.)
+              The projections are powered by Tile Service providers.  You can find examples at http://leaflet-extras.github.io/leaflet-providers/preview/
+             + */
+            L.tileLayer(
+              'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', 
+              leafletOptions
+            ).addTo(mymap)
+            mymap.setView(startingCoords,2)
+    
+            //Add the GeoJSON from the Manifest object
+            L.geoJSON(manifestObject.navPlace, {
+                pointToLayer: function (feature, latlng) {
+                  let appColor = "#08c49c"
+                  return L.circleMarker(latlng, {
+                    radius: 8,
+                    fillColor: appColor,
+                    color: "#000",
+                    weight: 1,
+                    opacity: 1,
+                    fillOpacity: 0.8
+                  })
+                },
+                onEachFeature: pointEachFeature
+            }).addTo(mymap)
+          }
+    
+          /**
+             A helper function for Leaflet.  Leaflet sees GeoJSON objects as "features".  This function says what to do with each feature when adding the feature to the map.  Here is where you detect what metadata appears in the pop-ups.  For our purposes, we assume the metadata you want to show is in the GeoJSON 'properties' property.  Our 'label' and 'summary' will be formatted as language maps, since they are most likely coming directly from a IIIF resource type and IIIF Presentation API 3 requires 'label' and 'summary' to be formatted as a language map.
+          */
+          function pointEachFeature(feature, layer){
+            //Oh no is the label you want in the Manifest? Instead of looking in the feature, look in the Manifest like... 
+            //popupContent += `<div><label>Label:</label>${manifest.label.en}</div>`
+            if (feature.properties) {
+              if(feature.properties.label.en) {
+                  popupContent += `<div><label>Label:</label>${feature.properties.label.en}</div>`
+              }
+              if(feature.properties.summary.en) {
+                  popupContent += `<div><label>Summary:</label>${feature.properties.summary.en}</div>`
+              }
+            }
+            layer.bindPopup(popupContent)
+          }
+        </script>
+    </html>
+    ```
+
+That's it! Now when this HTML page loads, it will initialize the Leaflet map with the FeatureCollection found in the `navPlace` property of this Manifest. Below you will see the code implemented! You should see a Leaflet load up, zoomed out pretty far, with a single greenish Point drawn in Germany. Click on the point to see your metadata!
```

Comment:
Suggested changes:

That's it! When the HTML page loads, it will initialize a Leaflet map with the FeatureCollection found in the `navPlace` property of the Manifest. In the live example below, you should see the leaflet map, zoomed out, with a single green point drawn in the center of Germany. Click on the point to see your metadata!